bdk-swift/README.md

```diff
@@ -50,3 +50,5 @@ own GitHub repository use the following steps:
 [Xcode]: https://developer.apple.com/documentation/Xcode
 [`bdk`]: https://github.com/bitcoindevkit/bdk
 [`bdk-ffi`]: https://github.com/bitcoindevkit/bdk-ffi
+
+Temporary line: needed to run CI
```

Comment:
Did you forgot to remove this line?

Reply:
I had a few more changes to the yaml files so I still needed them yesterday for my last push, but now that everything has passed I will remove them.